pkgs/development/python-modules/labelbox/default.nix

```diff
@@ -29,19 +28,37 @@
   typing-extensions,
 }:
 
-buildPythonPackage rec {
-  pname = "labelbox";
-  version = "6.6.0";
+let
+  version = "6.10.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.8";
-
   src = fetchFromGitHub {
     owner = "Labelbox";
     repo = "labelbox-python";
     tag = "v.${version}";
-    hash = "sha256-aMJJZ9ONnjFK/J4pyLTFQox/cC8ij85IYNlJTFrfV2I=";
+    hash = "sha256-EstHsY9yFeUhQAx3pgvKk/o3EMkr3JeHDDg/p6meDIE=";
+  };
+
+  lbox-clients = buildPythonPackage {
```

Comment:
please enable the test (maybe unittestCheckHook)

Reply:
Done. Thanks for catching this.